README.md

```diff
@@ -57,6 +57,52 @@ You probably have enabled "Secure Window" feature in SealNote settings.
 Disable it to be able to take screenshots or show window content in application
 switcher.
 
+Contributing
+--------------------------------
+
+Contributions are much appreciated. Please go through this section to understand how and what to contribute.
+
+If you need ideas for contribution, please check currently [open issues](https://github.com/vishesh/sealnote/issues). If you want to propose a new feature, please open a new issue. A good feature can be something which improves some workflow or security for a user.
+
+Before you start working on an issue, please claim it so that there is no duplication of effort. We don't assign an issue until you have a passing pull-request.
+
+### Tools of trade
+
+1. Java 1.7+
+2. Android API Level 19+
+
+### How to submit a pull request
+
+1. Clone sealnote:
+    `git clone https://github.com/vishesh/sealnote.git && cd sealnote`
+2. Verify that build succeeds:
+    `./gradlew assembleDebug`
+3. Create a topic branch:
+    `git checkout -b feature`
+4. **Make your changes.**
+5. Fork sealnote on GitHub.
+6. Add your fork as remote:
+    `git remote add <YOUR-USER> https://github.com/<YOUR-USER>/sealnote.git`
+7. Push to your fork:
+    `git push -u <YOUR-USER> HEAD`
+8. Open a pull request against **develop** branch describing your changes.
+
+### Style Guide
+
+The project follows [Google's Java Style Guide](https://google.github.io/styleguide/javaguide.html) with following exceptions:
+1. 4 spaces indentation instead of 2.
+2. Horizontal alignment is recommended if it makes code more readable.
+
+If you see any violation, feel free to submit a patch.
+
+### Commit guidelines
+
+There is no strict guideline on commit messages. However, wherever possible, follow [Git-book's](https://git-scm.com/book/ch5-2.html#Commit-Guidelines) commit guidelines for messages.
```

Comment:
whenever possible? Do you think this sentence sounds better without "commit guidelines for messages." at end?